core/ecschema-metadata/src/Metadata/KindOfQuantity.ts

```diff
@@ -234,7 +234,7 @@ export class KindOfQuantity extends SchemaItem {
 
     if (undefined !== this.presentationFormats) {
       const presUnitStrings = this.presentationFormats.map((format: Format | OverrideFormat) => {
-        if (format instanceof Format)
+        if (!OverrideFormat.isOverrideFormat(format))

```

Comment:
Can you use `isFormat` here instead of the looking if it's a Override?

Reply:
OverrideFormat does not subclass Format ... so an override format is not a format.

Reply:
okay, thanks